### PR TITLE
Ride details

### PR DESCRIPTION
**Pull request by ArunMRK**

### CODE REVIEW

File: Database/utils.py [hunk 1]
```diff
@@ -93,4 +93,72 @@ def extract_ride_hrt_rpm_power(message: str) -> dict:
...
-    return ride_dict
\ No newline at end of file
+    return ride_dict
+
+def list_average(ls: list) -> float:
+    return sum(ls) / len(ls)
+
+
+def current_ride_averages(current_ride_data: list) -> dict:
+    # Takes in readings over a ride and returns average values for rpm, heart rate, power and resistance over the course of the ride
```
Comment (bendg4444) [inline Database/utils.py:103]: Please Docstring """ COMMENT """ like so to conform to guide lines

File: Database/utils.py [hunk 2]
```diff
@@ -93,4 +93,72 @@ def extract_ride_hrt_rpm_power(message: str) -> dict:
...
+        'avg_rpm': avg_rpm_readings,
+        'avg_heart_rate': avg_heart_readings,
+        'avg_power': avg_power_readings,
+        'avg_resistance': avg_resistance_readings
+    }
+    return averages_dict
+
+
+def current_ride_maximums(current_ride_data: list) -> dict:
+    # Takes in readings over a ride, and returns max values for rpm, heart rate, power and resistance over the course of the ride
```
Comment (bendg4444) [inline Database/utils.py:127]: Same again

File: Database/utils.py [hunk 3]
```diff
@@ -93,4 +93,72 @@ def extract_ride_hrt_rpm_power(message: str) -> dict:
...
+        'avg_rpm': max_rpm_readings,
+        'avg_heart_rate': max_heart_readings,
+        'avg_power': max_power_readings,
+        'avg_resistance': max_resistance_readings
+    }
+    return maxima_dict
+
+
+def current_ride_timings(current_ride_data: list) -> dict:
+    # Takes in readings over a ride and returns duration, and start and end timestamps
```
Comment (bendg4444) [inline Database/utils.py:151]: same

**Review by bendg4444 — CHANGES_REQUESTED** [review]

File: Database/utils.py (hunk 1 above)
Comment (justas-b) [inline Database/utils.py:103]: proper docstrings next time please

**Pull request merged**